iati/core/resources/test_data/202/valid_dateorder.xml

```diff
@@ -4,5 +4,9 @@
   <planned-disbursement type="1">
    <period-start iso-date="2014-01-01" /><!-- Dates are in the correct order -->
    <period-end iso-date="2014-12-31" />
+   <period-start iso-date="2017-01-01" /><!-- Dates are in the correct order -->
```

Comment:
# MAJOR PROBLEM

This should cause the `{'less': 'planned-disbursement/period-start/@iso-date', 'more': 'planned-disbursement/period-end/@iso-date'}` test to fail - the start date of `2017-01-01` is after the end date of `2014-12-31`.